mbf_abstract_nav/src/abstract_execution_base.cpp

```diff
@@ -63,11 +63,6 @@ bool AbstractExecutionBase::start()
     thread_.join();
   }
 
-  // reset the member vars
-  cancel_ = false;
```

Comment:
We never reuse right now the execution class, so this is fine.

However, I think we should make the class "reusable" (or prevent explicitly the users from reusing an instance). This might be a cool application for our preRun and postRun methods:

```
virtual void preRun() {
// clear the old state:
outcome_ = 255;
message_.clear();
}
```
and
```
virtual void postRun(){
// reset the cancel flag
cancel_ = false
}
```
What do you guys think?

Reply:
> We never reuse right now the execution class, so this is fine.
> 
> However, I think we should make the class "reusable" (or prevent explicitly the users from reusing an instance). This might be a cool application for our preRun and postRun methods:

The idea of pre/postRun was other, namely calling map-specific code before/after the execution. Clearing cancel flag is a bit out of place, imho. In any case, I propose to wait until we make the executions reusable to made that change.

if you are both ok with this PR, i'll merge it as it is